src/Raven.Server/Documents/Revisions/RevisionsStorage.cs

```diff
@@ -670,11 +673,46 @@ public class DeleteOldRevisionsResult
 
             var deleted = DeleteRevisionsInternal(context, table, lowerIdPrefix, collectionName, changeVector, lastModifiedTicks, result.PreviousCount, revisionsToDelete, result);
 
+
+            NotifyAboutConflictRevisionsIfNeeded(conflicted, nonPersistentFlags, deleted, lowerIdPrefix);
+
             IncrementCountOfRevisions(context, lowerIdPrefix, -deleted);
             result.Remaining = result.PreviousCount - deleted;
             return result;
         }
 
+        private void NotifyAboutConflictRevisionsIfNeeded(bool conflicted, NonPersistentDocumentFlags nonPersistentFlags, long deleted, Slice lowerIdPrefix)
+        {
+            var lowerId = lowerIdPrefix.ToString();
```

Comment:
first check if you need to notify, then perform string (and other operations)

Reply:
Done